plugins.json

```diff
@@ -27,7 +27,7 @@
   "no-use-extend-native": "dustinspecker",
   "node": "mysticatea",
   "prettier": "https://github.com/prettier/eslint-plugin-prettier#options",
-  "promise": "https://github.com/xjamundx/eslint-plugin-promise#RULENAME",
+  "promise": "https://github.com/xjamundx/eslint-plugin-promise/blob/master/docs/rules/RULENAME.md",
```

Comment:
Actually, this can now use the shorthand syntax of just `xjamundx` here.

Reply:
Cool, will push up this change. 👍